src/tests/CMakeLists.txt

```diff
@@ -38,14 +38,10 @@ endif()
 serac_add_tests( SOURCES       axom_smoketest.cpp
                  DEPENDS_ON    axom::inlet)
 
-if(PETSC_FOUND)
-    serac_add_tests(SOURCES       petsc_smoketest.cpp
-                    DEPENDS_ON    PkgConfig::PETSC gtest axom::slic
-                    NUM_MPI_TASKS 4)
-
-    if(SLEPC_FOUND)
-        serac_add_tests(SOURCES       slepc_smoketest.cpp
-                        DEPENDS_ON    PkgConfig::SLEPC gtest axom::slic
-                        NUM_MPI_TASKS 4)
-    endif()
-endif()
+serac_add_tests(SOURCES       mfem_petsc_smoketest.cpp
```

Comment:
Was there a discussion that PETSC and SLEPC were now a hard requirement?

Reply:
My understanding is that when mfem is built with `+petsc+slepc`, we will have full access to those libraries, so I figured there would be no point to guard whether we're building with petsc/ slepc. I could re-add them though.

That is, assuming this PR goes through. I'm having issues with slepc.

The motivation behind this PR as a whole was a discussion I had with @tupek2 and his summer student (I'm not sure what his github account is) during the bug fix day. They mentioned it would be much easier to use petsc if they could use the mfem wrappers.

Reply:
If we are allowing MFEM to handle the petsc/slepc build options (which is fine imo), we need to check if they are enabled in the MFEM build or make them a hard requirement in our builds.

They are currently variants in our Spack package recipe:

https://github.com/LLNL/serac/blob/242c2b7d402fa4e774b16f70d91224892d6189c9/scripts/spack/packages/serac/package.py#L60-L62

If someone turns off these variants then these unit tests would fail to build.

Reply:
That's a good point. I'll update this.